tracker/dmlc_tracker/opts.py

```diff
@@ -122,8 +144,13 @@ def get_opts(args=None):
                               ' but corresponds to archieve files that will be unziped locally,' +
                               ' You can use this option to ship python libraries.' +
                               ' Only valid in yarn jobs.'))
-    parser.add_argument('--env', action='append', default=[],
-                        help='Client and ApplicationMaster environment variables.')
+    parser.add_argument('--env', type=str, default='',
+                        help='Comma separated list of Client and ApplicationMaster environment \
+                        variables.')
+    parser.add_argument('--env-server', type=str, default='',
+                        help='Comma separated list of environment variable:value for server only')
+    parser.add_argument('--env-worker', type=str, default='',
+                        help='Comma separated list of environment variable:value for worker only')
```

Comment:
Let's document that `--env-server` and `--env-worker` are only valid for the SSH tracker.